### PR TITLE
Denormalization of documents on-the-fly upon querying

### PR DESCRIPTION
**Pull request by deepakunni3**
Closes GDEV-106 and GDEV-107.

This PR adds the following,
- Add option to embed/denormalize references when fetching records via the API
- Add routes for DataAccessCommittee and DataAccessPolicy
- Update example JSON
- Add docstrings
- Fix formatting

### CODE REVIEW

File: examples/data_access_committees.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        },
+        {
+            "id": "DAC:0000002",
+            "title": "NFDI Consortium",
+            "dsecription": "The Nationale Forschungsdateninfrastruktur (NFDI) Consortium",
+            "main_contact": "abc@nfdi.de",
+            "has_members": ["abc@nfdi.de", "def@nfdi.de"]
+        }
+    ]
+}
```
Comment (ac-jorellanaf) [inline examples/data_access_committees.json:18]: Is EditorConfig not working? Or I guess you're not installing the VSCode extension?

**Review by ac-jorellanaf — APPROVED** [review]
Code seems in order.

File: examples/data_access_committees.json (hunk 1 above)
Comment (deepakunni3) [reply examples/data_access_committees.json]: Thanks for pointing that out @ac-jorellanaf ! Yes, I was missing the EditorConfig plugin.

Got it working and the new-lines are added to the JSON examples.

**Review by ac-jorellanaf — APPROVED** [review]

**Review by KerstenBreuer — APPROVED** [review]
I don't feel like the right person to review this. But I liked what I understood 😄

**Pull request merged**